lib/tasks/sw.rake

```diff
@@ -60,6 +58,38 @@ namespace :sw do
     harvester.harvest_from_wos_id_cap_profile_id_report args[:path_to_report]
   end
 
+  desc 'Compare publications (IDs) returned by APIs for an Author identified by their name [last,first,middle]'
+  task :wos_publications_for_name, [:last, :first, :middle] => :environment do |_t, args|
+    fail "last name argument is required" unless args[:last].present?
+    fail "first name argument is required" unless args[:first].present?
+    author_name = ScienceWire::AuthorName.new(args[:last], args[:first], args[:middle])
+    attribs = ScienceWire::AuthorAttributes.new(author_name, '', [], ScienceWireHarvester.new.default_institution)
+    puts "Querying ScienceWire for #{author_name.inspect}"
+    ids = ScienceWire::HarvestBroker.new(nil, ScienceWireHarvester.new).ids_from_dumb_query(attribs)
```

Comment:
Is the `ids_from_dumb_query` now a private method that requires a `send` here?

Reply:
Could this rake task use one instance of the `ScienceWireHarvester.new` (it's currently instantiating two of them).

Reply:
That hasn't been merged yet, so isn't yet required.  Could use the same Harvester, but I think it is useful that they don't have to be the same (`default_institution` could be a class method).  I'll say again I think `ScienceWire::HarvestBroker.new`'s 2nd param should take `ScienceWireHarvester.new` as a default also.